ssn/rdf/ontology/extensions/sosa-system.ttl

```diff
@@ -0,0 +1,119 @@
+  """@en ;
+  rdfs:subClassOf skos:Concept ;
+  rdfs:isDefinedBy sosa-systems: .
+
+## A short but extensible list of operating condition kinds is provided in SSN-System, such as normal operating conditions, suboptimal conditions, and survivable conditions.
+
+sosa:NormalOperatingConditions rdfs:subClassOf sosa:OperatingConditionKind ;
+  rdfs:label "Normal Operating Conditions"@en ;
+  skos:definition """
+  The conditions under which a system is expected to operate normally.
+  """@en ;
+  rdfs:isDefinedBy sosa-systems: .
+
+sosa:SuboptimalOperatingConditions rdfs:subClassOf sosa:OperatingConditionKind ;
+  rdfs:label "Suboptimal Conditions"@en ;
+  skos:definition """
+  The conditions under which a system is expected to operate suboptimally, but may recover if the conditions are restored to normal operating conditions.
+  """@en ;
+  rdfs:isDefinedBy sosa-systems: .
+
+sosa:SurvivableConditions rdfs:subClassOf sosa:OperatingConditionKind ;
+  rdfs:label "Survivable Conditions"@en ;
+  skos:definition """
+  The conditions under which a system is expected to survive. The system may be damaged if the survivable conditions are violated.
+  """@en ;
+  rdfs:isDefinedBy sosa-systems: .
+
+## Relationship involved in the description of the System capability
+
+sosa:hasSystemCapability a owl:ObjectProperty ;
+  schema:domainIncludes sosa:System , sosa:Sensor, sosa:Actuator, sosa:Sampler ;
+  schema:rangeIncludes sosa:ExecutionCollection , sosa:Execution ;
+  rdfs:label "has system capability"en ;
+  skos:definition "Relation between a System and an ObservationCollection (or Observation) describing the capability of the system. A sensor or system may have more than one set of system capabilities that are related to the conditions under which they hold true."@en ;
+  rdfs:isDefinedBy sosa-systems: .
+  
+
+## Detailed classes and relationships involved in the description of the conditions (typically an observation collection about the system environment) under which an observation or property (typically about the system capabilities) is valid
+
+sosa:inCondition a owl:ObjectProperty ;
+  schema:domainIncludes sosa:Observation , sosa:Property ;
+  schema:rangeIncludes sosa:ExecutionCollection , sosa:Execution ;
+  rdfs:label "in condition"@en ;
+  skos:definition "Describes the conditions (typically an observation collection about the system environment) defining the validity context of an observation (typically about the system capabilities) or a property (typically about the system capabilities)."@en ;
+  rdfs:isDefinedBy sosa-systems: .
+
+
+## Battery
+sosa:Battery rdfs:subClassOf sosa:System ;
+  rdfs:label "Battery"@en ;
+  skos:definition """
+  A Battery powering a System as its primary or secondary power source. As this is a specialization of the System class modeling of integrated battery management systems is also possible.
+  """@en ;
+  rdfs:isDefinedBy sosa-systems: .
+
+ssn:BatteryLifetime a owl:Class ;
```

Comment:
make it an instance, and move it to the system-capabilities-properties vocabulary